net/freeradius/src/opnsense/service/templates/OPNsense/Freeradius/proxy.conf

```diff
@@ -0,0 +1,128 @@
+{% if helpers.exists('OPNsense.freeradius.general.fallbackproxy') and OPNsense.freeradius.general.fallbackproxy == '1' %}
+proxy server {
+        default_fallback = yes
+}
+{% else %}
+proxy server {
+        default_fallback = no
+}
+{% endif %}
+{% if helpers.exists('OPNsense.freeradius.proxy.homeservers.homeserver') %}
+{%   for homeserver_list in helpers.toList('OPNsense.freeradius.proxy.homeservers.homeserver') %}
+{%     if homeserver_list.enabled == '1' %}
+home_server {{ homeserver_list.name }} {
+        type = {{ homeserver_list.type }}
+{%         if homeserver_list.addresstype == 'ipv4' %}
+        ipaddr = {{ homeserver_list.ipaddr }}
+{%         endif %}
+{%         if homeserver_list.addresstype == 'ipv6' %}
+        ipv6addr = {{ homeserver_list.ipaddr6 }}
+{%         endif %}
+{%         if homeserver_list.addresstype == 'virtual_server' %}
```

Comment:
Did you miss this one?

Reply:
No, only on line 22, I did a wrong call, it check virtual_server  on line 21 to insert homeserver_list.virtualserver on line 22.
My error was that instead of homeserver_list.virtualserver I worte homeserver_list.virtual_server